src/templates/actor/parts/matrix/SpritePowerList.html

```diff
@@ -2,6 +2,8 @@
     {{> 'systems/shadowrun5e/dist/templates/common/List/ListHeader.html'
             name=(localize 'SR5.Labels.ActorSheet.SpritePowers')
             itemId='sprite_power'
+            itemType='sprite_power'
+            itemData=(SpiritPowerItemData 'standard')
```

Comment:
looks like this says "SpiritPowerItemData". do they use the same thing or is this a bug?